airbyte-server/src/test/resources/migration/dummy_data/config/STANDARD_DESTINATION_DEFINITION/22f6c74f-5699-40ff-833c-4a879ea40133.json

```diff
@@ -3,5 +3,10 @@
   "name": "BigQuery",
   "dockerRepository": "airbyte/destination-bigquery",
   "dockerImageTag": "0.2.0",
-  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/bigquery"
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/bigquery",
+  "spec": {
+    "documentationUrl": "https://docs.airbyte.io/integrations/destinations/bigquery",
+    "connectionSpecification": {},
+    "supportsIncremental": true
+  }
```

Comment:
The rest of the changes in this PR look just like this. Adding this `spec` field to these files was necessary to get the RunMigrationTest to pass, as specs are now read from definitions instead of a specFetcher function.